Declarations/Interfaces.ts

```diff
@@ -134,6 +134,12 @@ export interface IBaseConfig {
     * Live Metrics custom host
     */
     quickPulseHost: string;
+    /**
+     * Enable web snippet auto html injection, default to false
+     */
+     enableAutoWebSnippetInjection: boolean;
+     isDebugWebSnippet: boolean;
```

Comment:
What is the difference between debug and not debug one?

Reply:
debug use js version and none debug use min.js version. I modified it from previous PR